Index: app/src/main/java/com/example/myclient/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/myclient/MainActivity.java	(revision 01a66ffb2ac5dd3c2ae42fe2f7370e1a3668c335)
+++ app/src/main/java/com/example/myclient/MainActivity.java	(date 1617201634662)
@@ -30,7 +30,7 @@
 
     Button btnSignIn, btnReg;
     RelativeLayout root;
-
+// test
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
